pkg/controller/operconfig/operconfig_controller.go

```diff
@@ -190,15 +191,26 @@ func (r *ReconcileOperConfig) Reconcile(request reconcile.Request) (reconcile.Re
 		}
 	}
 
+	oldOperConfig := operConfig.DeepCopy()
```

Comment:
This is backwards; you need to mutate the `DeepCopy()` and leave the original one untouched - or else you touch the shared object in the cache.